test/browsersync-compat.js

```diff
@@ -5,33 +5,45 @@ var pathModule = require('path');
 var root = pathModule.resolve(__dirname, '..', 'testdata') + '/';
 var processImage = require('../lib/processImage');
 
+var serverPort = '9999';
+
 expect.use(require('unexpected-http'))
     .use(require('unexpected-image'))
     .use(require('unexpected-resemble'))
     .use(require('unexpected-sinon'))
     .use(require('magicpen-prism'))
-    .addAssertion('<string> to respond with <object|number>', function (expect, subject, value) {
-        var modifiedSubject = subject.replace(' ', ' http://localhost:9999');
+    .addAssertion('<string|object> to respond with <object|number>', function (expect, subject, value) {
```

Comment:
This would read or document itself better if it was a middle-of-the-rocket assertion, something like `expect('GET /foo', 'prepended with hostname', 'to yield response', 200)`